src/SDKs/EventHub/Management.EventHub/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure EventHubs Management Library")]
 [assembly: AssemblyDescription("Provides developers with a library to create and manage all Azure Event Hubs resources.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.0.1.0")]
```

Comment:
@v-Ajnava usually you increase AssemblyFileVersion, in this case increase AssemblyFileVersion and no need to increase AssemblyVersion